README.md

```diff
@@ -91,29 +91,6 @@ kermit [-h] [-v] [-d] [-c config] [-t title] [-w workdir] [-e command]
 [-e command] sets the command to execute in terminal
 ```
 
-## Key Bindings
-
-| Key                         | Action                         |
-| --------------------------- | ------------------------------ |
-| `ctrl-alt-[c]`              | copy to clipboard              |
-| `ctrl-alt-[v]`              | paste from clipboard           |
-| `ctrl-alt-[t]`              | open a new tab                 |
-| `ctrl-alt-[r]`              | reload configuration file      |
-| `ctrl-alt-[d]`              | load the default configuration |
-| `ctrl-alt-[q]`              | exit the terminal              |
-| `ctrl-alt-[k][up]`          | increase font size             |
-| `ctrl-alt-[j][down]`        | decrease font size             |
-| `ctrl-alt-[=]`              | reset font size to default     |
-| `ctrl-alt-[return]`         | open a new tab                 |
-| `ctrl-alt-[num]`            | switch to the tab number       |
-| `ctrl-alt-[l][right][pgup]` | switch to the next tab         |
-| `ctrl-alt-[h][left][pgdn]`  | switch to the previous tab     |
-| `ctrl-alt-[w][backspace]`   | close the selected tab         |
-
-• Key bindings (`ctrl-alt`) can be set to `ctrl-shift` with using the config file.
-
-• Default key bindings (`ctrl-alt`) might conflict with your desktop environments window shortcuts. To solve this issue, key bindings should be changed to `ctrl-shift`.
```

Comment:
I would rather not delete this and have the defaults in place (for not making breaking changes)

Reply:
Sure, but how do you want it implemented? I basically see these options:
* Have a `disable_default_bindings` option, that disables the bindings 
* Overwriting bindings when one with the same keys is found
* Overwriting bindings when one with the same action is found

Reply:
> Overwriting bindings when one with the same action is found

I think this would be the best.

Reply:
Done